اقتراح تعديل الملف

atefBB opened the pull request:
اقتراح تعديل الملف ومراجعة النص الأصلي للتثبت من الترجمة.

atefBB committed:
اقتراح تعديل الملف

اقتراح تعديل الملف ومراجعة النص الأصلي للتثبت من الترجمة.

adaroobi commented:
@atefBB 
هنالك عنوان مقترح من المصدر:
[WikiPedia](https://ar.wikipedia.org/wiki/%D9%86%D9%85%D8%B7_%D8%A8%D8%B1%D9%85%D8%AC%D8%A9)

atefBB commented:
سأرسل سؤالا لمجمع اللغة العربية لأرى ماذا سيكون رأيهم في الترجمة.
شكرا.

atefBB commented:
هذا جواب مجمع اللغة العربية على الشبكة على السؤال عن مصطلح Programming Paradigms
http://www.m-a-arabia.com/vb/showthread.php?t=20518

atefBB committed:
اقتراح تعديل على الملف

اقتراح تعديل على الملف، واستبدال مصطلح Programming Paradigms بما يناسب ترجمته إلى اللغة العربية.

adaroobi commented:
ممتاز، اشكرك على مجهودك
